bin/lib/updateIndex.js

```diff
@@ -0,0 +1,56 @@
+const fs = require('fs')
+const path = require('path')
+const cheerio = require('cheerio')
+const LDP = require('../../lib/ldp')
+const { URL } = require('url')
+const debug = require('../../lib/debug')
+
+const { compileTemplate, writeTemplate } = require('../../lib/common/template-utils')
+const { getAccountManager, loadConfig, loadUsernames } = require('./cli-utils')
+const { getName, getWebId } = require('../../lib/common/user-utils')
+const { initConfigDir, initTemplateDirs } = require('../../lib/server-config')
+
+module.exports = function (program) {
+  program
+    .command('updateindex')
+    .description('Update index.html in root of all PODs that haven\'t been marked otherwise')
+    .action(async (options) => {
+      const config = loadConfig(program, options)
+      const configPath = initConfigDir(config)
+      const templates = initTemplateDirs(configPath)
+      const indexTemplatePath = path.join(templates.account, 'index.html')
+      const indexTemplate = await compileTemplate(indexTemplatePath)
+      const ldp = new LDP(config)
+      const accountManager = getAccountManager(config, { ldp })
+      const usernames = loadUsernames(config)
```

Comment:
I would've done that myself, but the server is sometimes difficult to write for, because there are so many things that need to be set up. The CLI scripts are kind of extreme in this sense =/

If you have concrete ways of rewriting this to make it easier to read, I'm all ears, but until then I'll leave it as it is.